dsl/src/test/scala/com/crobox/clickhouse/dsl/JoinQueryIT.scala

```diff
@@ -38,7 +39,7 @@ class JoinQueryIT
   ) { (joinType, result) =>
     it should s"join correctly on: $joinType" in {
       val query: OperationalQuery =
-        select(itemId).from(select(itemId).from(TwoTestTable).join(joinType, ThreeTestTable).using(itemId))
+        select(itemId).from(select(itemId).from(TwoTestTable).join(joinType, ThreeTestTable, Option("TTT")).using(itemId))
```

Comment:
How do we prevent that some creates an invalid query by having 2 join's with the same alias?

Reply:
We simply can't. Again, it's up to the implementation to provide unique names.... If omitted, random ones will be used which are 'safe'